core/src/commonMain/kotlin/com/ustadmobile/core/contentjob/JobConnectivityLiveData.kt

```diff
@@ -0,0 +1,37 @@
+package com.ustadmobile.core.contentjob
+
+import com.ustadmobile.core.networkmanager.ConnectivityLiveData
+import com.ustadmobile.door.DoorLiveData
+import com.ustadmobile.door.DoorMediatorLiveData
+
+class JobConnectivityLiveData(
```

Comment:
Class declaration should use Kotlin style (e.g. ending brackets should be on a new line)

Reply:
I think we should use the word mediator in the name of this class because that is it's purpose... e.g.

JobConnectivityLiveDataMediator